opentelemetry-exporter-geneva/geneva-uploader/src/ingestion_service/uploader.rs

```diff
@@ -37,6 +38,22 @@ impl From<GenevaConfigClientError> for GenevaUploaderError {
     }
 }
 
+impl From<reqwest::Error> for GenevaUploaderError {
+    fn from(err: reqwest::Error) -> Self {
+        let mut msg = format!("{:?}", err);
```

Comment:
Should we be using `Display` instead of `Debug` for more concise error messages?

`Debug` would print out a more verbose string with braces and field names and values.
` { kind: Request, url: "...", source: TimedOut }`

Reply:
`Display` doesn't print all details - specifically error kind (`ConnectionReset` in our case), let me see if we can print it explicitly, and then use `Display`

Reply:
Done, slightly complicated, but we can simplify it later.